.travis.yml

```diff
@@ -6,15 +6,26 @@ node_js:
   - "stable"
 
 sudo: false
+dist: trusty
+
+addons:
+  chrome: stable
 
 cache:
   directories:
-    - node_modules
-    - bower_components
+    - $HOME/.npm
 
 env:
   - TEST_FRAMEWORK=ember-cli-qunit
   - TEST_FRAMEWORK=ember-cli-mocha
+  # we recommend new addons test the current and previous LTS
+  # as well as latest stable release (bonus points to beta/canary)
+  - EMBER_TRY_SCENARIO=ember-lts-2.8
+  - EMBER_TRY_SCENARIO=ember-lts-2.12
+  - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-beta
+  - EMBER_TRY_SCENARIO=ember-canary
+  - EMBER_TRY_SCENARIO=ember-default
```

Comment:
Since this addon doesn't actually use anything from Ember (just from Ember-CLI), I'd rather not add all these additional builds to the matrix.

Reply:
I see, I'll remove these `EMBER_TRY_SCENARIO` scenarios.